src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -25,6 +25,21 @@ import {CDK_MENU, Menu} from './menu-interface';
 import {CdkMenuItem} from './menu-item';
 import {MenuStack, MenuStackItem, FocusNext} from './menu-stack';
 
+/**
+ * Check if the given element is part of the cdk menu module.
+ * @param target the element to check.
+ * @return true if the given element is part of the menu module.
+ */
+function isCdkMenuComponent(target: Element) {
+  const classList = target.classList;
```

Comment:
I think you could get away with `target.className.indexOf('cdk-menu') !== -1`

Reply:
Only check for `cdk-menu` and skip the others?

Reply:
They all start with `cdk-menu`, so as long as that prefix is there you're detecting them all

Reply:
My mistake, didn't see that it was `className` not `classList`